core/pom.xml

```diff
@@ -277,6 +277,11 @@ THE SOFTWARE.
       <version>1.8.3</version>
     </dependency>
     <dependency>
+        <groupId>org.apache.commons</groupId>
+        <artifactId>commons-compress</artifactId>
```

Comment:
The only dependency declarer in jenkinsci or is https://github.com/jenkinsci/p4-plugin. It declares the same version requirement, hence @p4paul can easily remove the dependency when he upgrades to the new core version.